src/styles/blocks/catalog/catalog.scss

```diff
@@ -0,0 +1,126 @@
+.catalog {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: repeat(4, 1fr);
+  place-items: center;
+  gap: $cards-horyzontal-gap $cards-vertical-gap;
+  grid-auto-rows: 1fr;
+
+  @include on-tablet {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @include on-mobile-l {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @include on-mobile-s {
+    grid-template-columns: 1fr;
+  }
+
+  &__card {
+    width: $card-width;
+    border: 1px solid $grey-300;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    transition: transform 300ms;
```

Comment:
consider ordering your css properties in a [better way](https://9elements.com/css-rule-order/)